src/plugins/vis_type_timelion/public/index.scss

```diff
@@ -0,0 +1,11 @@
+// EUI global scope
+
+@import '@elastic/eui/src/themes/eui/eui_colors_light';
+
+@import '@elastic/eui/src/global_styling/functions/index';
+@import '@elastic/eui/src/global_styling/variables/index';
+@import '@elastic/eui/src/global_styling/mixins/index';
```

Comment:
All of these imports can be removed when importing the manifest SASS file via the NP `plugin.ts`

Reply:
Done, thanks 👍